### PR TITLE
add vulkan-validationlayers/1.2.172

### PR DESCRIPTION
**Pull request by AndreyMlashkin**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by ericLemanissier — APPROVED** [review]

File: recipes/vulkan-validationlayers/all/conandata.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@ sources:
   "1.2.182":
     url: "https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/v1.2.182.tar.gz"
     sha256: "5a1f7027c06a8e5ae777d9053b5ce46f10ca623806a43332eb2da06fe46476d4"
+  "1.2.172":
+    url: "https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/v1.2.172.tar.gz"
+    sha256: "0e29bf2545658177d37052bcce09f29d3f464caf31d00a84c37368dd1a660d29"
```
Comment (SpaceIm) [inline recipes/vulkan-validationlayers/all/conandata.yml:7]: does it need the same patch than 1.2.182?

Comment (AndreyMlashkin) [reply recipes/vulkan-validationlayers/all/conandata.yml:7]: It worked for my purposes just out-of-the-box

Comment (SpaceIm) [reply recipes/vulkan-validationlayers/all/conandata.yml:7]: Well, it doesn't answer to my question. But looking at patches comments and related upstream PRs, 1.2.172 doesn't need patches.

**Comment by AndreyMlashkin** [comment]
I have the same error locally with the fix from another branch:

Undefined symbols for architecture x86_64:
  "BarrierOperationsType CoreChecks::ComputeBarrierOperationsType<VkBufferMemoryBarrier, VkImageMemoryBarrier>(CMD_BUFFER_STATE const*, unsigned int, VkBufferMemoryBarrier const*, unsigned int, VkImageMemoryBarrier const*) const", referenced from:
      CoreChecks::ValidateBarriers(core_error::Location const&, CMD_BUFFER_STATE const*, unsigned int, unsigned int, unsigned int, VkMemoryBarrier const*, unsigned int, VkBufferMemoryBarrier const*, unsigned int, VkImageMemoryBarrier const*) const in buffer_validation.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

Do someone know what is it about?

**Pull request closed without merging**